docs/java-rest/high-level/ml/delete-job.asciidoc

```diff
@@ -33,21 +35,24 @@ Defaults to `false`.
 ---------------------------------------------------
 include-tagged::{doc-tests-file}[{api}-request-wait-for-completion]
 ---------------------------------------------------
-<1> Use to set whether the request should wait until the operation has completed before returning.
-Defaults to `true`.
+<1> Use to set whether the request should wait until the operation has completed
+before returning. Defaults to `true`.
 
 
 [id="{upid}-{api}-response"]
-==== Delete Job Response
+==== Delete {anomaly-job} response
 
-The returned +{response}+ object indicates the acknowledgement of the job deletion or
-the deletion task depending on whether the request was set to wait for completion:
+The returned +{response}+ object indicates the acknowledgement of the job
+deletion or the deletion task depending on whether the request was set to wait
+for completion:
 
 ["source","java",subs="attributes,callouts,macros"]
 ---------------------------------------------------
 include-tagged::{doc-tests-file}[{api}-response]
 ---------------------------------------------------
-<1> whether was job deletion was acknowledged or not; will be `null` when set not to wait for completion
-<2> the id of the job deletion task; will be `null` when set to wait for completion
+<1> whether was job deletion was acknowledged or not; will be `null` when set
```

Comment:
It's not a problem introduced by this PR, but there's a spurious "was" between "whether" and "job".

Also most of these callout notes begin with capital letters but `<1>` and `<2>` here don't.

Since the PR is green feel free to merge as-is and leave these typos for a future PR if you prefer.

Reply:
Thanks, @droberts195 I've put those fixes in https://github.com/elastic/elasticsearch/pull/44865